packages/web-react/src/components/Tooltip/TooltipContext.ts

```diff
@@ -0,0 +1,55 @@
+import { MiddlewareData, Placement } from '@floating-ui/react';
+import { HTMLProps, MutableRefObject, createContext, useContext } from 'react';
+
+type TooltipContextType = {
+  anchorRef: ((node: HTMLElement | null) => void) & ((node: HTMLElement | null) => void);
```

Comment:
This type is here 3 times so it can be extracted and reused.

Reply:
done